pep-0586.rst

```diff
@@ -574,7 +562,7 @@ involving Literal bools. For example, we can combine ``Literal[True]``,
    @overload
    def is_int_like(x: Union[int, List[int]]) -> Literal[True]: ...
    @overload
-   def is_int_like(x: Union[str, List[str]]) -> Literal[False]: ...
+   def is_int_like(x: object) -> Literal[False]: ...
```

Comment:
Sadly mow mypy will give `error: Overloaded function signatures 1 and 2 overlap with incompatible return types`

Reply:
Return type should be `bool`.